tests/integration/cli_test.py

```diff
@@ -5,9 +5,9 @@
 import sys
 from operator import attrgetter
 
-from mock import patch
 from six import StringIO
 
+from .. import mock
```

Comment:
In a codebase that was a lot bigger with more potential name clashes/locations I can see the value. Given this size of codebase and `patch` is a common/well known mocking function, I think I'd err on side of explicit importing.